### ISSUE TITLE
how can we start fix this?

### ISSUE THREAD

**Issue by NguyenUoc98** [njd2/p#2]
Since when did you find the error on our site?

**Comment by NguyenUoc98** [comment]
We started hang your ads on out site, we will follow your request and hope you will keep your promise

**Comment by NguyenUoc98** [comment]
We aim for as little damage as possible and are willing to donate if you want

**Comment by njd2** [comment]
Thanks. As long as you keep your promise, I have no reason to harm you.
First, to fix the admin page, please focus on module title data and remove javascript from it.

**Comment by NguyenUoc98** [comment]
Why do you need traffic from our site? If it's for affiliate purposes only, we can donate to you accordingly

**Comment by njd2** [comment]
Yes. Affiliate purposes only. But what amount you can donate?

**Comment by NguyenUoc98** [comment]
We offer a starting price of $1000 for you to delete all crawled data and help us identify security flaws.

**Comment by NguyenUoc98** [comment]
You agree not to affect our business however you have edited 473,691 our user information making it impossible for them to continue on the site

**Comment by njd2** [comment]
No. I change it before the agreement because you delete my ads link multi-time. When we agree, I do not interact with your website. Do you have any backup? You can write scripts to match names in old record to new record. Or change to default names.

**Comment by njd2** [comment]
$1000 is good but I'm participating in a black hat hacker community contest. This content requires control of as many sites as possible, in order to hang ads for as long as possible. Is it possible to give time to think? The fact that I help you is certain because you have no benefit for me other than traffic. And I also somewhat feel guilty for attacking your website. However, this offer is worth considering because of the competition.

**Comment by njd2** [comment]
I agree with your offer. Please send 4 BNB to Metamask Address 0x4BE9B0D6a5831a3D49EA5C88f3ecCc8de4076E78 first. After I received BNB, will wait for 4-6 hours, **I will perform a penetration test (at this time, I only focus on Critical bugs which can gain access to your website, and database, not all bugs on your website, so I need do it again to list all) for your website again and list all vulnerabilities** (vul name, endpoint, impact, how to fix). When done, I will upload an excel file here.

**Comment by njd2** [comment]
Why did you remove my ads link?

**Comment by njd2** [comment]
I found you removed my ads link, why?

**Comment by njd2** [comment]
I think I have explained it clearly. Well, as you want, I will attack you again! Because you remove my ads link!

**Comment by NguyenUoc98** [comment]
We still place your ads on the site, you can use the traffic monitoring tool to check We agree to donate but need preparation time because our area is not available. As soon as we send 4 BNB to Metamask Address 0x4BE9B0D6a5831a3D49EA5C88f3ecCc8de4076E78 we will remove your ads

**Comment by NguyenUoc98** [comment]
You just need ads to be placed on the homepage or all pages?

**Comment by njd2** [comment]
Home page is the best selection.

**Comment by njd2** [comment]
You can buy BNB on Binance.

**Comment by NguyenUoc98** [comment]
We need 3-4h to buy BNB, please wait patiently for us, we really don't want you to vandalize or make data public. We can cooperate later. You can help us find security bugs and we'll donate to it

**Comment by NguyenUoc98** [comment]
We have transferred 0.01 BNB to the address you provided. Please confirm so we can continue sending the rest

**Comment by njd2** [comment]
Yes. Received 0.0095 BNB.

**Comment by NguyenUoc98** [comment]
We have sent the rest.  Now we will remove your ad and you need to upload the file detailing the security errors to us

**Comment by njd2** [comment]
Yes. Please wait for it.

**Comment by njd2** [comment]
Firstly I send you which vulnerability I used to dump the database, upload the shell, and gain access to your website. I removed my web shell on your server. I deleted your dumped database. I deleted the credentials of the user. Pls, give me your email I will send all bugs when possible. it took longer than i thought.

**SQL Injection**
Technical detail:
https://owasp.org/www-community/attacks/SQL_Injection
Prevention:
https://cheatsheetseries.owasp.org/cheatsheets/SQL_Injection_Prevention_Cheat_Sheet.html
End-point: https://123docz.net/documents/ajax/aja_send_doc_not_login.php?&docId=&email=email
Parameters: docId, email

GET /documents/ajax/aja_send_doc_not_login.php?&docId=&email=email HTTP/1.1
X-Requested-With: XMLHttpRequest
Referer: https://123docz.net/
Cookie: PHPSESSID=r961ruhcg538948s06j30no6ho; current-url=https%3A//123docz.net/document/11197255-dap-an-bai-kiem-tra-cuoi-hoc-ky-100-cau.htm; SEARCH_SUBMIT=%7B%22last_search%22%3A1664976490669%2C%22times%22%3A3%7D; ezTOC_hidetoc=1; ISSD_GUEST=guest_5161117; SHOW_EVENT_POPUP=1; PHPSESS1ON_V=MTExNzAzMg%3D%3D; check_coockie=check; scd=1000; CART_PENDING=%5B%5D
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Encoding: gzip,deflate
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4512.0 Safari/537.36
Host: 123docz.net
Connection: Keep-alive

**Comment by njd2** [comment]
Refer more: https://null-byte.wonderhowto.com/how-to/use-sql-injection-run-os-commands-get-shell-0191405/

**Comment by njd2** [comment]
Many endpoints in XSS, and LFI so take time to list them.

**Comment by NguyenUoc98** [comment]
Please send more to dev.123doc@gmail.com
